noodles/run/scheduler.py

```diff
@@ -117,15 +121,12 @@ def run(self, connection: Connection, master: Workflow):
                     pass
 
             if self.verbose:
-                print("sched result [{0}]: ".format(self.key_map[job_key]),
-                      result,
-                      file=sys.stderr, flush=True)
+                logger.info("sched result [{0}]: {1}".format(self.key_map[job_key], result))
+                for handler in logger.handlers:
+                    handler.flush()
 
             del self.jobs[job_key]
             if len(self.jobs) == 0 and graceful_exit:
-                for error in errors:
-                    print("Exception of type", type(error), ":")
-                    print(error)
```

Comment:
All exception info is already included in the preceding `logger.error` call, so no need to repeat it here.